src/date-range-picker.js

```diff
@@ -29,23 +30,26 @@ export var TinyDatePicker = TDP;
  * @param {HTMLElement} input The input associated with the datepicker
  * @returns {DateRangePickerInst}
  */
-export function DateRangePicker(container) {
+export function DateRangePicker(container, opts) {
+  opts = opts || {};
   var emitter = Emitter();
   var root = renderInto(container);
   var hoverDate;
   var state = {
     start: undefined,
     end: undefined,
   };
-  var start = TDP(root.querySelector('.dr-cal-start'), {
+  opts.startOpts = cp(opts.startOpts || {}, {
     mode: 'dp-permanent',
```

Comment:
I think I'd prefer not to mutate opts. I'll accept the PR, but I'll probably change this. Other than that, this is a great change. Thanks for putting this together!